packages/dart/noports_core/test/srvd/relay_authentication_escr_test.dart

```diff
@@ -0,0 +1,134 @@
+import 'dart:async';
+import 'dart:io';
+import 'package:at_chops/at_chops.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:noports_core/src/srv/relay_authenticators.dart';
+import 'package:noports_core/src/srvd/relay_auth_verifiers.dart';
+import 'package:test/test.dart';
+import 'package:uuid/uuid.dart';
+
+class MockRelayAuthVerifyHelper extends Mock implements RelayAuthVerifyHelper {}
+
+class MockSocket extends Mock implements Socket {}
+
+class MockStreamSubscription<T> extends Mock implements StreamSubscription<T> {}
+
+void main() {
```

Comment:
Tests for the Encrypted Signed Challenge-Response authentication and verification